### PR TITLE
Task: Arrays

### PR DESCRIPTION
**Pull request by osechka333**

### CODE REVIEW

File: Exercises/arrays/arrays.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//1. Make a function which will generate array with random length and random values.
+function generateArray() {
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js:2]: consider using `Array.from`

File: Exercises/arrays/arrays.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  for (let i = 0; i < arrayValues.length; i++) {
+    arrayValues[i] = Math.floor(Math.random() * 10);
+  }
+  return arrayValues;
+}
+//console.log(generateArray());
+//2. Make a function which will take array and return copy of it.
+function arrayCopy(array) {
+  const results = [...array];
+  return results;
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: just `return [...array]`

File: Exercises/arrays/arrays.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+//console.log(arrayCopy([1,2]));
+//3. Make a function which will take array and element and return a index of element in array.
+function array(arr, element) {
+  const results = arr.indexOf(element);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: no need to use plural here

File: Exercises/arrays/arrays.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  return results === -1 ? 'Element is not found' : results;
+}
+//console.log(array([1,2,3], 7));
+//4. Make function which will take two arrays and return association of them.
+function associationArray(array1, array2) {
+  const results = array1.concat(array2);
+  return results;
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: just `return arrayOne.concat(arrayTwo)` also not good to use numbers in variables

**Review by lizarusi — CHANGES_REQUESTED** [review]

File: Exercises/arrays/arrays.js (hunk 1 above)
Comment (osechka333) [reply Exercises/arrays/arrays.js:2]: thanks for comments! Updated and created new PR

**Review by Mike-Savin — APPROVED** [review]

**Pull request merged**